Fixing external link anchor to specification

simonvarey opened the pull request:
#### Summary
I fixed the anchor on the link to the CSS2 specification on https://developer.mozilla.org/en-US/docs/Web/CSS/frequency to `#frequencies`.

#### Motivation
The link to the CSS2 specification links to the anchor `#q19.0` which does not exist. 

#### Metadata
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

simonvarey committed:
Fixing external link anchor to specification

The link to the CSS2 specification links to the anchor `#q19.0` which does not exist. I have fixed the anchor to `#frequencies`.

teoli2003 commented on files/en-us/web/css/frequency/index.md:
```diff
@@ -50,7 +50,7 @@ Invalid frequency values:
 {{Specifications}}
 
-> **Note:** This data type was initially introduced in [CSS Level 2](https://www.w3.org/TR/CSS2/aural.html#q19.0) for the now-obsolete [aural](/en-US/docs/Web/CSS/@media/aural) [media type](/en-US/docs/Web/CSS/@media#media_types), where it was used to define the pitch of the voice. However, the `<frequency>` data type has been reintroduced in CSS3, though no CSS property is using it at the moment ...
+> **Note:** This data type was initially introduced in [CSS Level 2](https://www.w3.org/TR/CSS2/aural.html#frequencies) for the now-obsolete [aural](/en-US/docs/Web/CSS/@media/aural) [media type](/en-US/docs/Web/CSS/@media#media_types), where it was used to define the pitch of the voice. However, the `<frequency>` data type has been reintroduced in CSS3, though no CSS property is using it at the  ...
```
Thanks! This comment doesn't bring anything; we should remove the whole note.
```suggestion
```

wbamberg reviewed (dismissed):
👍 thank you for the fix, @simonvarey !

simonvarey committed:
Update files/en-us/web/css/frequency/index.md

Would it be worth adding a note just that no CSS property is using this property at the moment? Someone stumbling across this page might wonder what the property is for.

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

simonvarey commented:
@teoli2003 Would it be worth adding a note just that no CSS property is using this property at the moment? Someone stumbling across this page might wonder what the property is for.

teoli2003 commented:
> @teoli2003 Would it be worth adding a note just that no CSS property is using this property at the moment? Someone stumbling across this page might wonder what the property is for.

It isn't necessary, the browser compat section is clear about it.
